rollup.config.js

```diff
@@ -4,23 +4,21 @@ import graphql from '@rollup/plugin-graphql';
 import localResolve from 'rollup-plugin-local-resolve';
 import nodeResolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
-import pkg from './package.json';
+import { visualizer } from 'rollup-plugin-visualizer';
 
-let FORMAT = process.env.FORMAT;
+const FORMAT = process.env.FORMAT;
 
-// graphql-tools currently has a rollup build failure, so always call it an external until they fix it
-// otherwise, make all npm production dependencies external, plus their subpath usages
-// throughout the codebase, which rollup doesn't automatically pick up on
-let external = FORMAT==='es' ?
```

Comment:
Not sure why we were added these external dependancies only for ESM modules. I have removed the condition, so the following dependancies will be added as external for commonJs and UMD modules as well. @silentsakky thoughts ?

Reply:
I am not very sure why it was done that way. we might need to understand how these package systems work

Reply:
Will do some research on this.

Reply:
Tried using microbundle however will need more time to fix the issues like bundling the graphql files. Hence I am reverting this change for now so that it will behave like old build. Means the external dependancies will be added for only es modules.